openstack_dashboard/api/nova.py

```diff
@@ -258,10 +258,10 @@ def flavor_get(request, flavor_id, get_extras=False):
 
 
 @profiler.trace
-@memoized.memoized
-def flavor_list(request, is_public=True, get_extras=False):
+def flavor_list(request, is_public=True, get_extras=False, detailed=False):
```

Comment:
flavor_list defaults to `detailed=True`, setting it to False here breaks default behavior for the instances table.

additionally, memoized is removed now?